### DISCUSSION TITLE
cypress-multi-reporters and cypress-mochawesome-reporter

### DISCUSSION THREAD

**Discussion by diogormendes** [LironEr/cypress-mochawesome-reporter#118] [Q&A] [answered]
Just using **mochawesome** with "inlineAssets": true works as expected.

If I add cypress-multi-reporters inline is not working.

It creates the html but is dependent on the other folders.

Here is how is configured.

![Screenshot 2022-11-30 at 11 44 21](https://user-images.githubusercontent.com/17742979/204825915-f13a774f-e4b1-4c94-b325-be5e3a81543e.png)

![Screenshot 2022-11-30 at 12 05 15](https://user-images.githubusercontent.com/17742979/204833537-24f34926-2bc7-4f6a-a024-c88d17701c5e.png)

**Comment by LironEr** [comment, marked as answer]
Change `reporterOptions` to `cypressMochawesomeReporterReporterOptions`

See the example https://github.com/LironEr/cypress-mochawesome-reporter/blob/master/examples/multiple-reporters-config-file/reporter-config.json

**Comment by diogormendes** [comment]
I was using `mochawesomeReporterReporterOptions`

Thanks a lot @LironEr
